services/dedup/setup.go

```diff
@@ -23,3 +28,12 @@ func GetInstance(clearDB *bool) DedupI {
 
 	return dedupManager
 }
+
+func cleanUpV2Data() {
```

Comment:
I am having hard time understanding the use of this function. Can you please explain it's utility?